satellite/console/consoleweb/server.go

```diff
@@ -91,7 +93,41 @@ func (s *Server) appHandler(w http.ResponseWriter, req *http.Request) {
 	http.ServeFile(w, req, filepath.Join(s.config.StaticDir, "dist", "public", "index.html"))
 }
 
-// appHandler is web app http handler function
+// accountActivationHandler is web app http handler function
+// TODO: add some auth token in request header to prevent unauthorized token creation
+func (s *Server) createRegistrationTokenHandler(w http.ResponseWriter, req *http.Request) {
```

Comment:
I think we need to add authorization before we merge this, otherwise we'll end up exposing our production satellite.